webapp/src/components/kanban/kanbanCard.tsx

```diff
@@ -81,18 +102,53 @@ const KanbanCard = React.memo((props: Props) => {
         setShowConfirmationDialogBox(true)
     }
 
+    const isOnboardingBoard = board.title === 'Welcome to Boards!'
```

Comment:
Same here, you are adding a lot of logic here in a component that maybe can be extracted anywhere else, maybe move all this logic inside the `OpenCardTourStep` and `CopyLinkTourStep` components.

Reply:
Done.